frontend/public/components/utils/simple-tab-nav.tsx

```diff
@@ -31,7 +35,10 @@ export const SimpleTabNav: React.FC<SimpleTabNavProps> = ({
     <div>
       <Tabs
         onSelect={handleTabClick}
-        className={additionalClassNames}
+        className={classnames(
+          { 'pf-v6-u-mb-lg co-m-pane__body--offset-padding': withinSidebar },
```

Comment:
```suggestion
          { 'pf-v6-u-mb-lg pf-c-tabs--offset-padding': withinSidebar },
```
since what you're overriding is `pf-c-tabs` and put this rule in `_overrides.scss`.

Reply:
But on second look, maybe we don't bleed the tabs to the edges?  That's more in line with what PatternFly does.

Reply:
Good point! The less CSS the better :p